src/views/OnChainProperty.vue

```diff
@@ -13,6 +14,19 @@
 		  		@click.native="toast('Copied to clipboard')"
```

Comment:
This can be handled by 1 LOC.
`@click= "twitter ? navigateToTwitter : toast"`

Reply:
The only problem with this is that the click method is within the b-tooltip which contains the v-clipboard copy directive. I couldn't find a way to conditionally render that directive. In that case it would both open the new tab and copy the label to the clipboard, which might be OK?

Reply:
ah I guess right now I would just open a new tab and for copy label to the clipboard could be expressed by another button? Because anyway we have there copy link to user profile, so I would omit then copy to clipboard:)